services/QuillLMS/spec/models/activity_session_spec.rb

```diff
@@ -356,12 +356,12 @@
     context 'when unit_activity has a due date' do
       let(:unit) { create(:unit)}
       let(:classroom_unit) { create(:classroom_unit, unit: unit) }
-      let(:unit_activity) { create(:unit_activity, unit: unit, due_date:  Date.today+10.days) }
+      let(:unit_activity) { create(:unit_activity, unit: unit, due_date:  Date.current+10.days) }
```

Comment:
Can the examples in this file be `10.days.from_now` to be consistent with other clean-up you've done around time math in this PR?